.gitignore

```diff
@@ -45,7 +45,6 @@ Session.vim
 /_tmp/
 *.un~
 .vagrant
-!vendor/**/zz_generated.*
```

Comment:
Is this related to this PR or just something we missed when we removed vendor?

Reply:
Removed.